index.md

```diff
@@ -119,7 +119,7 @@ As long as you're in the first year, you are a part of the Science and Humanitie
 ***
 
 ### THE COURSE CONTENT
-Students are divided into Physics and Chemistry cycles, usually based on alphabetical order of their names. But there are some common subjects in each semester irrespective of the cycle. If you get the Physics cycle in Sem 1, you'll get Chemistry cycle in Sem 2 and vice versa.
+Students are divided into Physics and Chemistry cycles, usually based on alphabetical order of their names. But there are some common subjects in each semester irrespective of the cycle. If you get the Physics cycle in Sem 1, you'll get Chemistry cycle in Sem 2 and vice versa. The final letter determines the sem: A for odd sems and B for even sems.
```

Comment:
Don't worry about this change, I'll make a separate section in the article about course codes.